test/serializer/abstract/ResidualVoid.js

```diff
@@ -0,0 +1,8 @@
+let x = global.__abstract ? __abstract("number", "42") : 42;
+y = global.__residual ? __residual("void", function() {
+  // Prepack doesn't (yet) partially evaluate a loop over an abstract value,
```

Comment:
Yes this was left over from the original test case that I refactored, I will change this!